docs/container-image-builds.md

```diff
@@ -0,0 +1,39 @@
+# Container Builds
+
+All container images Foreman builds will live at `quay.io/foreman/$service:$tag`.
+Dependent services that we do not build ourselves will have a copy of containers stored in Foreman’s quay.io to allow tagging with the Foreman version.
+
+There will be three flavors of Foreman containers:
+
+  * Vanilla Foreman
+  * Foreman + Katello
+  * Foreman + all plugins
+
+Stage versions of containers will have stage in the name and be published in the Foreman's quay repository:
+
+  * `quay.io/foreman/$service-stage:$tag`
+
+The base image will be `quay.io/centos/centos:stream9`.
+
+## Containerfile Artifacts
+
+Container files will be stored in repositories for each service that mimic packaging.
+
+  * foreman-oci-images
+  * pulpcore-oci-images
+  * candlepin-oci-images
```

Comment:
Should we create the repos already, even if they're empty? Then update this to be links to them

Reply:
Ehh, we can update when we are ready to start that.